app/src/main/kotlin/cz/covid19cz/erouska/ui/helpsearch/HelpSearchVM.kt

```diff
@@ -84,9 +84,11 @@ class HelpSearchVM @ViewModelInject constructor(
                 val q = SearchableQuestion(question.category)
                 q.answer = newA.first
                 q.question = newQ.first
-                searchResult.add(q)
+
+                withContext(Dispatchers.Main) { searchResult.add(q) }
```

Comment:
Proč je potřeba main thread tady?

Reply:
pretoze inak to pada na
`cz.covid19cz.erouska.dev E/AndroidRuntime: FATAL EXCEPTION: DefaultDispatcher-worker-2
    Process: cz.covid19cz.erouska.dev, PID: 32707
    android.view.ViewRootImpl$CalledFromWrongThreadException: Only the original thread that created a view hierarchy can touch its views.
        at android.view.ViewRootImpl.checkThread(ViewRootImpl.java:6903)
        at android.view.ViewRootImpl.requestLayout(ViewRootImpl.java:1048)
....
        at androidx.recyclerview.widget.RecyclerView$Adapter.notifyItemRangeInserted(RecyclerView.java:7498)
        at arch.adapter.BaseRecyclerAdapter$initOnListChangedListener$1.onItemRangeInserted(BaseRecyclerAdapter.kt:52)`